pkg/metrics/controller_test.go

```diff
@@ -399,7 +396,6 @@ func TestControllerFleetReplicasCount_ResetMetricsOnDelete(t *testing.T) {
 func TestControllerFleetAutoScalerState(t *testing.T) {
 	runtime.FeatureTestMutex.Lock()
 	defer runtime.FeatureTestMutex.Unlock()
-	require.NoError(t, runtime.ParseFeatures(fmt.Sprintf("%s=false", runtime.FeatureResetMetricsOnDelete)))
```

Comment:
Since this test was only going to pass when the false was false, the entire test can be deleted.

Reply:
I have deleted the functions.